R/format_data.R

```diff
@@ -1467,24 +1470,26 @@ fmt_date <- function(data,
     fns = list(
       default = function(x) {
 
-        # If `x` is of the `Date` type, simply make
-        # that a character vector
-        if (inherits(x, "Date")) {
+        # If `x` is of the `Date` or `POSIXct` types, simply transform
+        # `x` into a character vector
+        if (inherits(x, "Date") || inherits(x, "POSIXct")) {
           x <- as.character(x)
```

Comment:
I wouldn't do this conversion, just let strftime do it.

Reply:
Done.